docs/modules/ROOT/pages/how-tos/upgrade-v2-to-v3.adoc

```diff
@@ -0,0 +1,16 @@
+// NOTE: this page is a preparation for the next major release by removing deprecated features.
+// It shall only be linked in the nav once a major release is pending.
+
+= Upgrade from v2 to v3
+
+
+This guide describes the steps to perform an upgrade of the component from version 2.x to 3.x.
+
+Version 3.0 changes how secrets are managed and configured for the LDAP IdP.
+
+
+== Upgrade guide for LDAP provider
+
+With version 3.x, the parameter `openshift4_authentication.identityProviders.<name_of_the_provider>.ldap.bindPassword` as `string` was marked **deprecated** and has been removed with this release.
+
+See xref:references/deprecation-notice.adoc[Deprecated features] how to upgrade.
```

Comment:
Note: This page doesn't exist in this PR (see #33)

Reply:
This page is not required anymore, looking at #33, as you add the deprecation notice for v2.3.0 and not v3.0.0.

Reply:
Did you see the note at the top of this page? I left it purposefully there as preparation so it doesn't get forgotten on the next major release.